image-layout.md

```diff
@@ -70,7 +70,7 @@ Those tags will often be represented in an image-layout repository with matching
 ### Example Ref
 
 ```
-$ cat ./refs/v1.0 | jq
+$ cat ./refs/v1.0 | jq .
```

Comment:
At least with jq 1.5, the `.` filter seems to be the default, so the old versions worked fine.  I haven't checked to see if all recent versions of jq are like this or not.  If you need `jq .` but get an error on a bare `jq`, which version of jq are you running?